src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -33,54 +38,63 @@ public void clear() {
 
     @Test
     public void register_alreadyExistedUser_notOk() {
-        Storage.people.add(user);
+        storage.add(user);
         assertThrows(RuntimeException.class, () -> registrationService.register(user));
     }
 
     @Test
     public void register_nullUser_notOk() {
         user = null;
-        assertThrows(NullPointerException.class, () -> registrationService.register(user));
+        assertThrows(RuntimeException.class, () -> registrationService.register(user));
     }
 
     @Test
-    public void register_adult_notOk() {
+    public void register_underMinAge_notOk() {
         user.setAge(17);
         assertThrows(RuntimeException.class, () -> registrationService.register(user));
     }
 
     @Test
-    public void register_lengthOfPassword_notOk() {
+    public void register_lengthOfPasswordTooShort_notOk() {
         user.setPassword("sdf");
         assertThrows(RuntimeException.class, () -> registrationService.register(user));
     }
 
     @Test
     void register_nullLoginInUser_notOk() {
         user.setLogin(null);
-        assertThrows(NullPointerException.class, () -> registrationService.register(user));
+        assertThrows(RuntimeException.class, () -> registrationService.register(user));
+    }
+
+    @Test
+    void register_theSameUserWithDifferentAge_notOk() {
+        storage.add(user);
+        user.setAge(27);
+        assertThrows(RuntimeException.class, () -> registrationService.register(user));
     }
```

Comment:
when we write it this way the user in our storage will have same age as user that we register (you can check it in debugger), 
so let's create a new user with different age